vms/platformvm/txs/mempool/mempool.go

```diff
@@ -173,17 +188,22 @@ func (m *mempool) Add(tx *txs.Tx) error {
 	return nil
 }
 
-func (m *mempool) Has(txID ids.ID) bool {
-	return m.Get(txID) != nil
+func (m *mempool) Get(txID ids.ID) (*txs.Tx, bool) {
+	m.lock.Lock()
+	defer m.lock.Unlock()
+
+	return m.get(txID)
 }
 
-func (m *mempool) Get(txID ids.ID) *txs.Tx {
-	tx, _ := m.unissuedTxs.Get(txID)
-	return tx
+func (m *mempool) get(txID ids.ID) (*txs.Tx, bool) {
+	return m.unissuedTxs.Get(txID)
```

Comment:
I think we can put this in `Get` directly

Reply:
It's used on Add to avoid re-adding a tx. I think we need the non-locked unexported version. Unless we modify the hashmap to return a boolean (false if insertion is really an update sort of thing)